.github/workflows/e2e-tests.yml

```diff
@@ -22,6 +22,10 @@ jobs:
     permissions:
       packages: write
     steps:
+      - id: lower-repo
+        shell: pwsh
+        run: |
+          "::set-output name=repository::$($env:GITHUB_REPOSITORY.ToLowerInvariant())"
```

Comment:
It will give an error if the repository name has some uppercase letters (exactly what AKamyshnikova has).
The is a discussion here https://github.com/orgs/community/discussions/10553 and I used one of the suggested options to workaround this.